libindy/src/api/payments.rs

```diff
@@ -346,6 +381,8 @@ pub extern fn indy_register_payment_method(command_handle: CommandHandle,
                                            parse_get_txn_fees_response: Option<ParseGetTxnFeesResponseCB>,
                                            build_verify_payment_req: Option<BuildVerifyPaymentReqCB>,
                                            parse_verify_payment_response: Option<ParseVerifyPaymentResponseCB>,
+                                           sign_with_address: Option<SignWithAddressCB>,
```

Comment:
This is a breaking changes of API, so we should be very carefully and do it once and generic enough. I will put my "changes requested" to block the PR until enough feedback from different maintainers would be received.

Reply:
Does adding functionality cause breaking changes to an API? None of the existing functionality is changing at all, and none of the software that is currently using the API will break if this functionality is added.

Reply:
This particular call is changed, so it's formally breaking changes of API. But it affects only payment plugins, and Mike has fixed both known to us libnullpay and libsovtoken

Reply:
Ah, I understand. That makes sense.